afs/templates/views/components/reports/observation.htm

```diff
@@ -210,57 +222,57 @@ <h1 class="afs-summary-report-title"><span data-bind="text: displayname" class="
                 </div>
                 <!-- /ko -->
-                <!-- Procedure --> 
-                <!-- ko if: procedureSummary() && procedureSummary().length -->
-                <div class="afs-table pad-btm">
-                    <table class="table table-striped" cellspacing="0" width="100%">
-                        <thead>
-                            <tr class="afs-table-header">
-                                <th>{% trans "Procedure/Method Used" %}</th>
-                            </tr>
-                        </thead>
-                        <tbody data-bind="dataTablesForEach: { data: procedureSummary, dataTableOptions: getTableConfig(1) }">
-                            <tr>
-                                <td><a class="afs-table-link" data-bind="click: function() {window.open(link);}, text: displayValue" href="javascript:void(0)"></a></td>                            </tr>
-                        </tbody>
-                    </table>
+            </div>
+        </div>
+        <!-- Procedure --> 
+        <div class="afs-report-section">
+            <h2 class="afs-report-section-title">{% trans "Procedures" %}</h2>
+            <div class="afs-report-subsection" >
+                <!-- ko ifnot: procedureSummary() -->
+                <h3 class="afs-report-nodata-message">{% trans "There are no procedures for this resource" %}</h3>
+                <!-- /ko -->
+                <!-- ko if: procedureSummary() -->
+                <div class="afs-report-data-block">
+                    <div class="afs-block-value"><a data-bind="click: function() {window.open(link);}, text: displayValue" href="javascript:void(0)"></a></div>
                 </div>
                 <!-- /ko -->
-                <!-- Person Carried out --> 
-                <!-- ko if: personSummary() && personSummary().length -->
-                <div class="afs-table pad-btm">
-                    <table class="table table-striped" cellspacing="0" width="100%">
-                        <thead>
-                            <tr class="afs-table-header">
-                                <th>{% trans "Person Carrying Out Observation" %}</th>
-                            </tr>
-                        </thead>
-                        <tbody data-bind="dataTablesForEach: { data: personSummary, dataTableOptions: getTableConfig(1) }">
-                            <tr>
-                                <td><a class="afs-table-link" data-bind="click: function() {window.open(link);}, text: displayValue" href="javascript:void(0)"></a></td>
-                            </tr>
-                        </tbody>
-                    </table>
+            </div>
+        </div>
+        <!-- Person Carried out -->
+        <div class="afs-report-section">
+            <h2 class="afs-report-section-title">{% trans "Persons Carried out" %}</h2>
```

Comment:
This should be "Persons Carrying Out Observation" (remove s if this is cardinality 1)

Reply:
Thanks!